### PR TITLE
#2559. Representation variable cannot be augmented

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Augmentation-libraries/augmenting_getters_A01_t02.dart [hunk 1]
```diff
@@ -80,17 +80,14 @@ augment extension Ext {
   }
 }
 
-extension type ET(String instanceVariable) {
+extension type ET(String _) {
   static String staticVariable = "Original";
 }
 
-augment extension type ET(String instanceVariable) {
+augment extension type ET(String _) {
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/augmenting_getters_A01_t02.dart:87]: This will be a syntax error, shouldn't we expect an error somewhere?

**Review by eernstg — COMMENTED** [review]
One case needs to be adjusted, or an error should be expected.

**Comment by sgrekhov** [comment]
Updated. PTAL

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**